ubyssey/events/forms.py

```diff
@@ -47,6 +47,18 @@ def save(self, commit=True):
 
         facebook_image_url = self.data.get('facebook_image_url')
 
+        old_event_url = self.data.get('event_url')
+        old_ticket_url = self.data.get('ticket_url')
+
+        has_protocol_event_url = old_event_url.startswith('http://') or old_event_url.startswith('https://')
+        has_protocol_ticket_url = old_ticket_url.startswith('http://') or old_ticket_url.startswith('https://')
+
+        if not has_protocol_event_url:
+            event.event_url = "http://" + old_event_url
+
+        if not has_protocol_ticket_url:
+            event.ticket_url = "http://" + old_ticket_url
+
```

Comment:
This logic is good, but I think a helper function would make things a bit cleaner:

```py
def format_url(url):
  # code goes here

event.event_url = format_url(event.event_url)